### PR TITLE
Implement LIC 11

### PR DESCRIPTION
**Pull request by elmaxe**
This PR implements LIC 11 and test cases for it.

Fixes: #11

### CODE REVIEW

File: src/main/java/CMV.java [hunk 1]
```diff
@@ -158,7 +158,21 @@ private boolean LIC10() {
...
+    if (G_PTS < 1 || G_PTS > points.length - 2) return false;
+
+    for (int k = 0; k < points.length - 2 - G_PTS; k++) {
+      int i = k;
+      int j = k + G_PTS + 1;
+      Point a = points[i];
+      Point b = points[j];
+
+      if (a.x - b.x < 0) return true;
+      else continue;
```
Comment (K2017) [inline src/main/java/CMV.java:173]: `else continue;` is redundant here.

**Review by K2017 — APPROVED** [review]
Good job. Can decide to address my other comment or not it's not that important. I'll approve either way.

File: src/main/java/CMV.java (hunk 1 above)
Comment (elmaxe) [reply src/main/java/CMV.java]: Fixed in 169bd5f

**Review by Internet-Person-IP — APPROVED** [review]
looks good.

**Pull request merged**